.github/workflows/cicd.yml

```diff
@@ -37,3 +37,15 @@ jobs:
         with:
           push: ${{ github.event_name == 'push' }}
           tags: gombocai/cli:latest
+
+      - name: Run Trivy vulnerability scanner
+        uses: aquasecurity/trivy-action@0.24.0
+        with:
+          image-ref: 'docker.io/gombocai/cli:latest'
```

Comment:
Change to `gombocai/cli:latest` to be parallel to the build.

Reply:
Fixed